test/es-module/test-esm-invalid-pjson.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const { spawn } = require('child_process');
+const { strictEqual, ok } = require('assert');
+
+const entry = fixtures.path('/es-modules/import-invalid-pjson.mjs');
+const invalidJson = fixtures.path('/node_modules/invalid-pjson/package.json');
+
+const child = spawn(process.execPath, [entry]);
+child.stderr.setEncoding('utf8');
+let stderr = '';
+child.stderr.on('data', (data) => {
+  stderr += data;
+});
+child.on('close', common.mustCall((code, signal) => {
+  strictEqual(code, 1);
+  strictEqual(signal, null);
+  ok(
+    stderr.includes(
+      [
+        '[ERR_INVALID_PACKAGE_CONFIG]: ',
+        `Invalid package config ${invalidJson}, `,
+        'Unexpected token } in JSON at position 14'
```

Comment:
It seems this becomes `position 16` in windows resulting in the Windows test failure.

```suggestion
        `Unexpected token } in JSON at position ${isWin ? 16 : 14}`
```